env0/resource_environment_test.go

```diff
@@ -37,9 +37,10 @@ func TestUnitEnvironmentResource(t *testing.T) {
 
 	createEnvironmentResourceConfig := func(environment client.Environment) string {
 		return resourceConfigCreate(resourceType, resourceName, map[string]interface{}{
-			"name":        environment.Name,
-			"project_id":  environment.ProjectId,
-			"template_id": environment.LatestDeploymentLog.BlueprintId,
+			"name":          environment.Name,
+			"project_id":    environment.ProjectId,
+			"template_id":   environment.LatestDeploymentLog.BlueprintId,
+			"force_destroy": true,
```

Comment:
because now destroy requires "force_destroy" I had to add it to all the tests for test cleanup to work